src/components/YieldPools/ListItem.tsx

```diff
@@ -171,6 +179,7 @@ const ListItem = ({ farm, setSharedPoolAddress }: ListItemProps) => {
     isStakeInvalidAmount = true
   }
 
+  // This only use to validate in Stake modal
   const isStakeDisabled = isStakeInvalidAmount
```

Comment:
LOL. This is a redundant variable. haha

Reply:
its use to validate if user typein value is valid or not, quite legacy code, but if want to remove need to refactor the modals in this file

Reply:
I know this is a legacy code. I mean at this line: https://github.com/KyberNetwork/kyberswap-interface/pull/1659/files#diff-7a13446d6ec90d398824f960a761ccb4c3877e697337fcdb890df3c3ba5a2ef0R745, we can change it to 
```
disabled={isStakeInvalidAmount}
```

Reply:
ah ok :))))